Dockerfile.template

```diff
@@ -76,22 +127,32 @@ ENV REDMINE_DOWNLOAD_SHA256 {{ .sha256 }}
 ENV RAILS_LOG_TO_STDOUT true
 
 RUN set -eux; \
-# if we use wget here, we get certificate issues (https://github.com/docker-library/redmine/pull/249#issuecomment-984176479)
-	curl -fL -o redmine.tar.gz "$REDMINE_DOWNLOAD_URL"; \
+	wget -O redmine.tar.gz "$REDMINE_DOWNLOAD_URL"; \
 	echo "$REDMINE_DOWNLOAD_SHA256 *redmine.tar.gz" | sha256sum -c -; \
 	tar -xf redmine.tar.gz --strip-components=1; \
 {{ if .version | IN("5.0.10", "5.1.5") then ( -}}
 	# https://www.redmine.org/issues/42113 (aka https://github.com/rails/rails/issues/54260)
 	# 5.1: https://github.com/redmine/redmine/commit/c7b1f00fc1b42fd9f77b8e6574dae453ced642b4
 	# 5.0: https://github.com/redmine/redmine/commit/f27570120b7a672249bfebfe4d62da506785e146
+{{ if is_alpine then ( -}}
+	apk add --no-cache patch; \
+{{ ) else ( -}}
+	savedAptMark="$(apt-mark showmanual)"; \
```

Comment:
Similar to `savedAptMark`, should we add `--virtual` here? :thinking:

I don't think this is super important.

Reply:
I guess this is along the same lines as "is BusyBox's `patch` enough?" (and similarly unimportant)